### PR TITLE
Use Go 1.6 vendoring for all packages

### PR DESCRIPTION
**Pull request by paulmey**
Used godep to save the dependencies under the vendor directory. Can now build with Go 1.6 without setting GO15VENDOREXPERIMENT environment variable.

### CODE REVIEW

**Comment by paulmey** [comment]
@boumenot @ahmetalpbalkan is this what vendoring has come to? Everybody is just supposed to copy in kilobytes of someone else's source code?

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ go:
   - 1.6
 
 env:
-  - GO15VENDOREXPERIMENT=0
+  - GO15VENDOREXPERIMENT=1
```
Comment (ahmetalpbalkan) [inline .travis.yml]: you might as well drop `go 1.5` build?

**Comment by ahmetalpbalkan** [comment]
I heard IPv6 is going to fix everything.

**Comment by paulmey** [comment]
what's up with `go build -v ./packer/...` not giving any output in the build? seems like it's cheating?

File: .travis.yml (hunk 1 above)
Comment (paulmey) [inline .travis.yml]: Well, we say that we support building with 1.5... I like offering people a choice, but yes, its completely arbitrary

**Pull request merged**